internal_ws/yksg/src/lib.rs

```diff
@@ -191,6 +192,7 @@ struct StackFrame {
 pub struct StopgapInterpreter {
     /// Active stack frames (most recent last).
     frames: Vec<StackFrame>,
+    returnval: bool,
```

Comment:
Did we agree on `rv`for a field elsewhere? I can't remember, but seems to ring a bell.

Reply:
I don't remember this.

Reply:
OK. Well I prefer "rv" or "return_val", but don't feel all that strongly.

Reply:
Fixed in 7e160bbd13c050265ff1e5a8292f471e209dffd6.